README.md

```diff
@@ -36,7 +36,7 @@ deploy-code:
 
     steps:
       - name: Download artifacts from release
-        uses: im-open/download-release-asset@v1.0.2
+        uses: im-open/download-release-asset@v1.1.0
```

Comment:
It doesn't look like you've added a commit message that will bump the minor version.

Reply:
It needs to be in a commit message. `git-version-lite` only looks at the commit log.